api/src/settings/settings.py

```diff
@@ -25,6 +25,9 @@
 insecure_key = "insecure"
 SECRET_KEY = os.getenv("SECRET_KEY", insecure_key)
 
+# The token that is allowed to post data to protected endpoints
+AUTHORIZATION_TOKEN = os.environ['AUTHORIZATION_TOKEN']
+
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = SECRET_KEY == insecure_key
```

Comment:
Heeft relatie met mn comment over de secret_key. Ik stel voor dat we deze ook default false zetten en naar een environment variable trekken: ```DEBUG = os.getenv('DEBUG', False)```